docs/includes/pairlists.md

```diff
@@ -89,6 +90,30 @@ be caught out buying before the pair has finished dropping in price.
 
 This filter allows freqtrade to ignore pairs until they have been listed for at least `min_days_listed` days.
 
+#### OffsetFilter
+
+Offsets an incoming pairlist by a given `offset` value.
+
+As an example it can be used in conjunction with `VolumeFilter` to remove the top X volume pairs. Or to split
+a larger pairlist on two bot instances.
+
+Example to remove the first 10 pairs from the pairlist:
+
+```json
+"pairlists": [{
+        "method": "OffsetFilter",
+        "offset": 10
+}],
```

Comment:
```suggestion
"pairlists": [
    {
        "method": "OffsetFilter",
        "offset": 10
    }
],
```
Putting the brackets at new lines makes it easier to copy/paste the sample (you won't / can't use only Offsetfilter - so copying the whole sample is not possible.